README.md

```diff
@@ -206,12 +206,12 @@ var publicKey = fs.readFileSync('/path/to/public.pub');
 app.use(jwt({ secret: publicKey }));
 ```
 
-RSA algorithm is also supported by providing jwks-rsa.expressJwtSecret callback in the
+RSA algorithm is also supported by providing jwks promise provider in the
```

Comment:
Could we rephrase this to:



If the `secret` option is a function, this function is called for each JWT received in order to determine which secret is used to verify the JWT.

The signature of this function should be ` ({ alg, kid }) => [Promise(secret)]`, where `alg` and `kid` are the algorithm and key id fields from the decoded JWT header.

This option can be used to support JWKS (JSON Web Key Set) providers by using [node-jwks-rsa](https://github.com/auth0/node-jwks-rsa). For example:


```javascript
const { koaJwtSecret } = require('jwks-rsa');

app.use(jwt({
    secret: koaJwtSecret({
        jwksUri: 'https://sandrino.auth0.com/.well-known/jwks.json',
        cache: true,
        cacheMaxEntries: 5,
        cacheMaxAge: ms('10h'),
    }),
    audience: 'http://myapi/protected',	
    issuer:   'http://issuer'
});
```